modules/doc/content/application_development/profiling.md

```diff
@@ -7,6 +7,64 @@ installation process.  Although it works best on Linux platforms, it also
 works reasonably well on Mac OS.  Instruments also works well for profiling
 applications on Mac systems.
 
+## Install Google's gperftools
+
+If your environment package from the MOOSE installation process has included Google's gperftools,
+you can skip this step.
+To install Google's gperftools on your own, our recommended procedure is to
+first install `libunwind`:
+
+```
+cd $HOME
+git clone git@github.com:libunwind/libunwind.git
+cd libunwind
+./configure --prefix=$PWD/installed
+make -j<num_procs>
```

Comment:
May explain that `num_procs` is the number of threads you want to use

Reply:
I changed this to `make -j$MOOSE_JOBS`. I am open to changing it something else as well